ci/cudf_pandas_scripts/run_tests.sh

```
@@ -81,34 +81,47 @@ python -m pytest -p cudf.pandas \
+        pandas_requirements_txt="pandas-compat-${version}-requirements.txt"
+        rapids-dependency-file-generator \
+            --config dependencies.yaml \
+            --file-key test_cudf_pandas_compat \
+            --output requirements \
+            --matrix "cuda=${RAPIDS_CUDA_VERSION};pandas_compat_version=${version}" \
+            > "${pandas_requirements_txt}"
+
+        env_name="venv_pandas_${version}"
+        python -m venv --clear "${env_name}"
+        # shellcheck disable=SC1090
+        source "${env_name}/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./bin/activate was not specified as input (see shellcheck -x).

(SC1091)